tests/python/unittest/test_arith_iter_affine_map.py

```diff
@@ -107,32 +106,20 @@ def test_fuse():
     res = tvm.arith.detect_iter_map([y * 4 + x], var_dom([(x, 3), (y, 4)]))
     assert len(res) == 0
 
-    # simple stride pattern
-    res = tvm.arith.detect_iter_map([x * 4 + y * 2], var_dom([(x, 3), (y, 2)]))
-    assert len(res) == 1
```

Comment:
check if these testcases are intentionally deleted, i see they are moved, just want to make sure